Stellar-operation-create-withdrawal-request.x

```diff
@@ -29,52 +33,75 @@ struct CreateWithdrawalRequestOp
 };
 
 /******* CreateWithdrawalRequest Result ********/
-
+//: CreateWithdrawalRequest result codes
 enum CreateWithdrawalRequestResultCode
 {
-	// codes considered as "success" for the operation
-	SUCCESS = 0,
-
-	// codes considered as "failure" for the operation
-	INVALID_AMOUNT = -1, // amount is 0
-	INVALID_CREATOR_DETAILS = -2, // external details size exceeds max allowed
-	BALANCE_NOT_FOUND = -3, // balance not found
-	ASSET_IS_NOT_WITHDRAWABLE = -4, // asset is not withdrawable
-	CONVERSION_PRICE_IS_NOT_AVAILABLE = -5, // failed to find conversion price - conversion is not allowed
-	FEE_MISMATCHED = -6, // expected fee does not match calculated fee
-	CONVERSION_OVERFLOW = -7, // overflow during converting source asset to dest asset
-	CONVERTED_AMOUNT_MISMATCHED = -8, // expected converted amount passed by user, does not match calculated
-	BALANCE_LOCK_OVERFLOW = -9, // overflow while tried to lock amount
-	UNDERFUNDED = -10, // insufficient balance to perform operation
-	INVALID_UNIVERSAL_AMOUNT = -11, // non-zero universal amount
-	STATS_OVERFLOW = -12, // statistics overflowed by the operation
-	LIMITS_EXCEEDED = -13, // withdraw exceeds limits for source account
-	INVALID_PRE_CONFIRMATION_DETAILS = -14, // it's not allowed to pass pre confirmation details
-	LOWER_BOUND_NOT_EXCEEDED = -15, //amount to withdraw is too small 
+    // codes considered as "success" for the operation
+    //: CreateWithdrawalRequest operation successfully applied
+    SUCCESS = 0,
+
+    // codes considered as "failure" for the operation
+    //: Trying to create withdrawal with amount of 0
+    INVALID_AMOUNT = -1,
+    //: Creator details either are not valid JSON
+    INVALID_CREATOR_DETAILS = -2,
+    //: Source balance to withdraw from not found 
+    BALANCE_NOT_FOUND = -3, // balance not found
+    //: Asset cannot be withdrawn, as AssetPolicy::WITHDRAWABLE is not set
+    ASSET_IS_NOT_WITHDRAWABLE = -4,
+    //: Deprecated
+    CONVERSION_PRICE_IS_NOT_AVAILABLE = -5, // failed to find conversion price - conversion is not allowed
+    //: Fee provided in the operation and fee calculated on operation application are mismatched
```

Comment:
Fee charged on the operation and fee calculated on the operation fulfillment are mismatched